afids_utils/afids.py

```diff
@@ -54,6 +54,43 @@ class AfidPosition:
     z: float = attrs.field()
     desc: str = attrs.field(validator=_validate_desc)
 
+    def __sub__(self, other: AfidPosition) -> tuple[float, float, float]:
```

Comment:
I am open to either, though I think I lean towards the former if we enforce that subtraction of the objects has to be between the same label. I don't think this would prevent anyone from computing the distance between other fiducials, but would require a more manual approach.